README.md

```diff
@@ -133,6 +135,11 @@ Example, filtering by object name:
 /v1/{type}?filter=metadata.name="can-be-a-substri"
 ```
 
+When SQLite caching is enabled, there are different operators.  `FIELD = VALUE` does exact matching,
+`FIELD ~ VALUE` does partial matching. You can use quotes around `VALUE` to delimit it; if it contains
+only alphanumeric characters and characters that aren't significant in filters (like '=', ',', etc.),
+they don't need to be quoted.
+
```

Comment:
Maybe an example of this would help.  I'm sure it's correct, but I'm not totally clear.

Reply:
I added examples for all the operators when sql is enabled. Might as well doc them here, and then the docs team can run with it if they so desire.